docs/framework/windows-workflow-foundation/samples/using-the-invokemethod-activity.md

```diff
@@ -4,9 +4,9 @@ ms.date: "03/30/2017"
 ms.assetid: b6643550-d043-4ac7-8069-9c55ebbb4233
 ---
 # Using the InvokeMethod Activity
-This sample demonstrates how to use the <!--zz <xref:System.Activities.Statements.InvokeMethod%601> -->[<code>System.Activities.Statements.InvokeMethod\`1</code>](https://msdn.microsoft.com/library/dd647677.aspx) activity to invoke public methods in public classes. The <!--zz <xref:System.Activities.Statements.InvokeMethod%601> -->[<code>System.Activities.Statements.InvokeMethod\`1</code>](https://msdn.microsoft.com/library/dd647677.aspx) activity allows a workflow to call methods against objects, pass in parameters, get the return value, specify types for generic methods, and specify whether the method is synchronous or asynchronous. 
+This sample demonstrates how to use the <xref:System.Activities.Statements.InvokeMethod%601> activity to invoke public methods in public classes. The <xref:System.Activities.Statements.InvokeMethod%601> activity allows a workflow to call methods against objects, pass in parameters, get the return value, specify types for generic methods, and specify whether the method is synchronous or asynchronous. 
```

Comment:
This a tricky change and probably will be more involved. This topic should probably be deleted. The sample doesn't exist and System.Activities.Statements.InvokeMethod<T> was removed at some point apparently. My suggestion would be to delete this file and add an entry to https://docs.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/samples/built-in-activities in the https://github.com/dotnet/docs/blob/master/.openpublishing.redirection.json file. Let us know if you need any help with this,